Collector_2019-08-05.sql

```diff
@@ -0,0 +1,57 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 192.168.20.20 (MySQL 5.6.44)
+# Database: Collector
+# Generation Time: 2019-08-05 12:12:03 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table garden plants
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `garden plants`;
+
+CREATE TABLE `garden plants` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `plant_name` varchar(255) NOT NULL DEFAULT '',
+  `plant_type` varchar(255) NOT NULL DEFAULT '',
+  `position` varchar(255) NOT NULL DEFAULT '',
+  `soil_type` varchar(255) NOT NULL DEFAULT '',
+  `colour` varchar(255) DEFAULT NULL,
+  `cost` float unsigned NOT NULL,
```

Comment:
Does an unsigned float make sense here?

Reply:
Here's a video that might explain better than I can: https://www.youtube.com/watch?v=PZRI1IfStY0

Reply:
Thanks. As discussed it makes sense to be unsigned as it should always be a positive number. Re being a float - I take your point that it might be better as an integer.

Reply:
Agreed for a first pass, maybe something to take another look at when refining the data once the app is completed